tests/kafkatest/benchmarks/core/benchmark_test.py

```diff
@@ -152,8 +156,11 @@ def test_long_term_producer_throughput(self, compression_type="none", security_p
         self.logger.info("\n".join(summary))
         return data
 
+    @cluster(num_nodes=5)
     @parametrize(security_protocol='SSL', interbroker_security_protocol='PLAINTEXT')
-    @matrix(security_protocol=['PLAINTEXT', 'SSL', 'SASL_PLAINTEXT', 'SASL_SSL'], compression_type=["none", "snappy"])
+    @matrix(security_protocol=['PLAINTEXT', 'SSL'], compression_type=["none", "snappy"])
+    @cluster(num_nodes=6)
```

Comment:
Seems unfortunate that we need to split the SASL cases everywhere and and annotate the cluster size twice. I'm tempted to even say that it might be worth just wasting the extra node in those cases to reduce the need for this.